.github/workflows/run-test-manual-new.yml

```diff
@@ -0,0 +1,186 @@
+name: NEW Run test manually 
+# This workflow is for testing only and will be disabled and removed after testing
```

Comment:
question: so we are not going to keep this and remove it in next PR?

Reply:
> question: so we are not going to keep this and remove it in next PR?

@witoszekdev If it works as I expect I will replace the old workflow for `Run test manually` with this new one in the next PR.